github-for-jira.sd.yml

```diff
@@ -204,6 +204,22 @@ alarms:
     HighSeverityAlarmWhenTooManyELB5xxErrors: null
     LowSeverityAlarmWhenTooManyBackend5xxErrors: null
     LowSeverityAlarmWhenTooManyELB5xxErrors: null
+  eventsQueueSize:
```

Comment:
This is a queue alert, we need to put it inside each queue. It wouldn't have any effect on this level (And probably fail the deployment)

Also, don't forget to validate the config using `atlas micros validate` command (You'll need to set all env variables we have in this file to make this validation work)

Reply:
Tried running this but it doesn't work anymore. Not sure what the updated version is.